packages/astro/src/cli/add/index.ts

```diff
@@ -641,13 +641,22 @@ async function tryToInstallIntegrations({
 }): Promise<UpdateResult> {
 	const installCommand = await getInstallIntegrationsCommand({ integrations, cwd });
 
+	const inheritedFlags = Object.entries(flags)
+		.map(([flag, value]) => {
+			if (flag == '_') return;
+
+			return [`-${flag}`, value.toString()];
+		})
+		.filter(Boolean)
+		.flat();
```

Comment:
I don't think we should pass everything down. To be safe, we can scope to `--save-dev`, `-D` etc only. In case we support more options in the future, this part won't be glossed over.

Reply:
I don't get why we should only pass those down. It's essentially giving back the normal functionality with flags to the cli. It will work as pnpm without the `astro add` wrapper. Please let me know if I am missing something

Reply:
I think `astro add` shouldn't be seen as `pnpm add` but smarter. It does a lot more like editing configs, adding integration-specific configs etc. The installation phase is one of the steps for `astro add`. So passing the args directly to that single step is a bit off to me.

Reply:
I agree with @bluwy that we should only support `--save-dev` and `-D` for now.